Add changelog for release v0.38.0 (#2114)
elastisys/compliantkubernetes-apps: changelog/0.38.md

# v0.38.0

Released 2024-04-30

## Release highlights

- Gatekeeper upgrade to v3.15.1

## Changes by kind

### Feature(s)

> - [#1943](https://github.com/elastisys/compliantkubernetes-apps/pull/1943) - Add tekton-pipelines controller `Ayoub Ed-dafali`

Reviewer: :slightly_smiling_face: 
```suggestion
- [#1943](https://github.com/elastisys/compliantkubernetes-apps/pull/1943) - Add tekton-pipelines controller `Ayoub Ed-dafali`, [@davidumea](https://github.com/davidumea)
```

> - [#1999](https://github.com/elastisys/compliantkubernetes-apps/pull/1999) - add azure storage support to apps [@vomba](https://github.com/vomba)

Reviewer 2: ```suggestion
- [#1999](https://github.com/elastisys/compliantkubernetes-apps/pull/1999) - Add azure storage support to apps [@vomba](https://github.com/vomba)
```

- [#2013](https://github.com/elastisys/compliantkubernetes-apps/pull/2013) - Add rclone restore helm chart [@aarnq](https://github.com/aarnq)
- [#2060](https://github.com/elastisys/compliantkubernetes-apps/pull/2060) - Support Azure DNS and VXLAN node network policy [@vomba](https://github.com/vomba)

> - [#2078](https://github.com/elastisys/compliantkubernetes-apps/pull/2078) - Added kured alert for when nodes have required reboot for 30 days [@Elias-elastisys](https://github.com/Elias-elastisys)

Reviewer 2: ```suggestion
- [#2078](https://github.com/elastisys/compliantkubernetes-apps/pull/2078) - Add kured alert for when nodes have required reboot for 30 days [@Elias-elastisys](https://github.com/Elias-elastisys)
```

Consistency 😄

> - [#2101](https://github.com/elastisys/compliantkubernetes-apps/pull/2101) - Add a gatekeeper constraint that warns application developers when they deploy with less then 3 replicas [@vomba](https://github.com/vomba)

Reviewer 3: ```suggestion
- [#2101](https://github.com/elastisys/compliantkubernetes-apps/pull/2101) - Add a gatekeeper constraint that warns application developers when they deploy with less then 2 replicas [@vomba](https://github.com/vomba)
```

Reviewer 2: ```suggestion
- [#2101](https://github.com/elastisys/compliantkubernetes-apps/pull/2101) - Add a gatekeeper constraint that warns application developers when they deploy with less than 2 replicas [@vomba](https://github.com/vomba)
```
